### DISCUSSION TITLE
Cfg File Commands

### DISCUSSION THREAD

**Discussion by ubermunchen** [apohl79/audiogridder#273] [Q&A]
First, I really want to thank you for undertaking this project. It's something I've been looking for for a very long time. 

Second, do you have any documentation on how to modify the server cfg file? I'm in windows, though I suppose people using other systems may want to know the same. At this time, I'm more particularly wondering your format for custom directories, including multiple directories. I see the options in the cfg. I just want to be sure that I modify it correctly.

**Comment by apohl79** [comment]
You could add a few dummy dirs via the AG settings dialog. After that you see the format in the file.

**Reply by ubermunchen** [reply]
That actually relates to one of the problems I believe I have, so maybe I need to post in the issues section. I have no gui on my cpus when I initially run the server. It just disappears into the background. I do have a gui on the slave and host when I load a plugin on the host, but it's only the plugin instance. I assume the dialog you're referring to would be in the gui at load.

**Reply by apohl79** [reply]
here is an example config:
```
{
    "ExcludePlugins": [],
    "ID": 0,
    "Logger": true,
    "NAME": "win10-dev",
    "ScanForPlugins": true,
    "ScreenCapturingFFmpeg": true,
    "ScreenCapturingFFmpegEncoder": "webp",
    "ScreenCapturingOff": false,
    "ScreenDiffDetection": true,
    "ScreenQuality": 0.8999999761581421,
    "Tracer": false,
    "UseJucePluginIDs": true,
    "VST": true,
    "VST2": true,
    "VST2Folders": [
        "C:\\Program Files\\VstPlugins2"
    ],
    "VST3Folders": [
        "C:\\Program Files\\Common Files\\VST3_2"
    ],
    "VSTNoStandardFolders": true
}
```

**Comment by ubermunchen** [comment]
Ok.... I found the menus. They're in the system tray on windows.

**Reply by kwalker99** [reply]
System Tray? I haven't see a tray app with the latest beta version, am i missing something?

**Reply by apohl79** [reply]
Windows is hiding tray icons from the taskbar automatically. Please check the tray menu.

**Reply by CaBuffalo** [reply]
And make sure you have the display scale setting set at 100%.
